Packs/VirusTotal/Integrations/VirusTotalV3/VirusTotalV3.yml

```diff
@@ -100,6 +100,90 @@ configuration:
   defaultvalue: '3'
   type: 0
   required: false
+- additionalinfo: 'Select the list of relationships which will be brought from the api. Some of the relationships are signed with * key which indicates that they are
```

Comment:
for all of this params. they will only be relevant for 6.2 TIM customers, correct? we need to think of a way to reflect that in the docs/additionalinfo.